### ISSUE TITLE
Confusing/missing build instructions

### ISSUE THREAD

**Issue by wincak** [maliit/keyboard#19]
I'm looking for a keyboard to be used in an embedded device and would like to try Maliit 2. However I'm having trouble compiling the project.

```
CMake Error at CMakeLists.txt:29 (find_package):
  By not providing "FindMaliitPlugins.cmake" in CMAKE_MODULE_PATH this
  project has asked CMake to find a package configuration file provided by
  "MaliitPlugins", but CMake did not find one.

  Could not find a package configuration file provided by "MaliitPlugins"
  (requested version 0.99.1) with any of the following names:

    MaliitPluginsConfig.cmake
    maliitplugins-config.cmake

  Add the installation prefix of "MaliitPlugins" to CMAKE_PREFIX_PATH or set
  "MaliitPlugins_DIR" to a directory containing one of the above files.  If
  "MaliitPlugins" provides a separate development package or SDK, be sure it
  has been installed.
```
And the "maliit/plugins" project's About states:

> Contains the reference input method plugins, such as the Maliit Keyboard.

Is it necessary to include Maliit keyboard to build a Maliit keyboard? Interestingly, if I try to build the plugins project itself it fails with exactly the same error message, like if it was trying to include itself.

Would it be possible to create a step-by-step guide how to build the keyboard on desktop Linux just to try it out?
The way the keyboard is split into several projects some of which seem to be abandoned is pretty confusing tbh, especially since the available documentation mentions old Qt4 and a non-existent wiki.

**Comment by jpetersen** [comment]
Yes sorry. You just need to build first the framework (which provides the mallit-plugins stuff) and second the keyboard.

I will try to add better build instrucstions soon.

**Comment by gkaklas** [comment]
For anyone finding this issue, if you have the nix package manager installed, I managed to have a shell with all the necessary dependencies and then build the project by running:

```shell
nix develop nixpkgs#maliit-keyboard
cmake -B build -S .
cmake --build build
```

(I found the build commands used in the [ArchLinux package](https://gitlab.archlinux.org/archlinux/packaging/packages/maliit-keyboard/-/blob/9e5d89e1f2f5fded72b75a4153182b5ada6dc10a/PKGBUILD))
